linear_operator/test/linear_operator_test_case.py

```diff
@@ -883,13 +883,13 @@ def _test_triangular_linear_op_inv_quad_logdet(self):
             linear_op = self.create_linear_op()
             rootdecomp = linear_operator.root_decomposition(linear_op)
 
-            if isinstance(rootdecomp, linear_operator.lazy.CholLinearOperator):
```

Comment:
Huh. Did this test never run? This should have failed, no?

Reply:
It seems to never have been executed because the method starts with an underscore ("_test_..."). Removing the underscore makes this execute, but fail.

Reply:
Ugh. OK how about we land this as is (with underscore) and create an issue for fixing the test in a separate PR? (unless this is a quick fix).

Reply:
![tests-cant-fail-5b5b40](https://user-images.githubusercontent.com/1605878/189153667-23164109-1e7e-46fa-a9b9-f2279a1cfda3.jpg)

Reply:
I'll spend a few more minutes on the test to see if I can figure it out quickly. It puzzles me at the moment.

Reply:
I understand the failures now, but did not fix it in this PR. Raised an [issue here](https://github.com/cornellius-gp/linear_operator/issues/11).